test-clients/src/main/java/com/hedera/services/bdd/suites/HapiApiSuite.java

```diff
@@ -109,72 +100,88 @@ public String name() {
 				: simpleName.substring(0, simpleName.length() - "Suite".length());
 	}
 
-	public enum FinalOutcome { SUITE_PASSED, SUITE_FAILED }
-	protected FinalOutcome finalOutcomeFor(List<HapiApiSpec> completedSpecs) {
-		return completedSpecs.stream().allMatch(HapiApiSpec::OK) ? SUITE_PASSED : SUITE_FAILED;
+	public List<HapiApiSpec> getFinalSpecs() {
+		return finalSpecs;
+	}
+
+	public boolean canRunAsync() {
+		return false;
+	}
+
+	public void skipClientTearDown() {
+		this.tearDownClientsAfter = false;
+	}
+
+	public void deferResultsSummary() {
+		this.deferResultsSummary = true;
+	}
+
+	public void summarizeDeferredResults() {
+		if (deferResultsSummary) {
+			deferResultsSummary = false;
+			summarizeResults(getResultsLogger());
+		}
 	}
 
 	public FinalOutcome runSuiteAsync() {
 		return runSuite(this::runAsync);
 	}
+
 	public FinalOutcome runSuiteSync() {
 		return runSuite(this::runSync);
 	}
 
+	protected FinalOutcome finalOutcomeFor(List<HapiApiSpec> completedSpecs) {
+		return completedSpecs.stream().allMatch(HapiApiSpec::OK) ? SUITE_PASSED : SUITE_FAILED;
+	}
+
 	private FinalOutcome runSuite(Consumer<List<HapiApiSpec>> runner) {
-		getResultsLogger().info("-------------- STARTING " + name() + " SUITE --------------");
+		if (!deferResultsSummary) {
+			getResultsLogger().info("-------------- STARTING " + name() + " SUITE --------------");
+		}
 		List<HapiApiSpec> specs = getSpecsInSuite();
-		specs.stream().forEach(spec -> spec.setSuitePrefix(name()));
+		specs.forEach(spec -> spec.setSuitePrefix(name()));
 		runner.accept(specs);
 		finalSpecs = specs;
 		summarizeResults(getResultsLogger());
-		HapiApiClients.tearDown();
+		if (tearDownClientsAfter) {
+			HapiApiClients.tearDown();
+		}
 		return finalOutcomeFor(finalSpecs);
 	}
 
 	public static HapiSpecOperation[] flattened(Object... ops) {
 		return Stream
 				.of(ops)
 				.map(op -> (op instanceof HapiSpecOperation)
-						? new HapiSpecOperation[] { (HapiSpecOperation)op }
-						: ((op instanceof List) ? ((List)op).toArray(new HapiSpecOperation[0]) : (HapiSpecOperation[])op))
+						? new HapiSpecOperation[] { (HapiSpecOperation) op }
+						: ((op instanceof List) ? ((List) op).toArray(
+						new HapiSpecOperation[0]) : (HapiSpecOperation[]) op))
```

Comment:
nit: it's probably the editor's formatting, it's probably easier to read with the original or the something like the following:
``` 
: ((op instanceof List) ? ((List)op).toArray(new HapiSpecOperation[0]) 
          : (HapiSpecOperation[])op))
```